webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-manager.js

```diff
@@ -143,6 +143,9 @@ class SakaiRubricsManager extends RubricsElement {
 
   sortRubrics(event) {
     const sortInput = event.target.className;
+    if (sortInput === "") {
```

Comment:
Why not get rid of that toString on sortInput. className is always a string. Also, you can just go if (!sortInput). Empty strings are falsy in JS.
https://developer.mozilla.org/en-US/docs/Web/API/Element/className